src/js/mixins/TabsMixin.js

```diff
@@ -74,9 +75,11 @@ const TabsMixin = {
         onClick={this.tabs_handleTabClick.bind(this, tab)}
         {...attributes}
       >
-        <span className="menu-tabbed-item-label-text">
-          {this.tabs_tabs[tab]}
-        </span>
+        <Trans
+          render="span"
+          className="menu-tabbed-item-label-text"
+          id={this.tabs_tabs[tab]}
```

Comment:
We're translating all labels independently. Do we need this `Trans`

Reply:
`i18nMark` doesn't translate, it just let's the extractor know about a string that needs translation. You then plug any of those into a Trans component to perform the actual translation.

Reply:
Oh! ok, thanks for explaining this